device/mellanox/x86_64-mlnx_msn4410-r0/thermal_policy.json

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/thermal_policy.json
```

Comment:
same here. please verify why 2700 is used and not 4700

Reply:
the thermal policy is also common for all the platforms.